app/models/user.rb

```diff
@@ -46,7 +46,7 @@ def approved
 
   def approved=(val)
     # ActiveAdmin use "1" to denote true
-    if val == "1"
+    if val == "1" || val == true
```

Comment:
FYI， you can use `ActiveRecord::Type::Boolean.new.type_cast_from_user("0") # false` to parse boolean value from request params